src/ontology/lexmatch/unmapped_ncit_lex.tsv

```diff
@@ -1,47 +1,614 @@
 subject_id	subject_label	object_id	predicate_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string
 ID		A oboInOwl:hasDbXref	>A oboInOwl:source							
+MONDO:0000376	respiratory system cancer	NCIT:C4571	MONDO:equivalentTo	Malignant Respiratory System Neoplasm	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	malignant respiratory system neoplasm
```

Comment:
Many of these new mappings are from NCIT oio:hasExactSynonym matches and this increased amount of these mappings is probably due to the past issue with synonyms not being maintained in the component ncit.owl file.